catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -268,6 +278,22 @@ public void setAttributeOverrides(List<String> attributeOverrides) {
         this.attributeOverrides = attributeOverrideMap;
     }
 
+    public void setBlacklistedFilePatterns(List<String> blacklistedFilePatterns) {
```

Comment:
Changed the CDM to use the `bad.file.extensions` and `bad.files` properties instead